src/Domain/PasswordRecovery/Mailer.php

```diff
@@ -16,24 +17,34 @@
 class Mailer
 {
     private MailerInterface $mailer;
+
```

Comment:
whitespace much?

Reply:
my local autoformatter... maybe we should put some more rules into `.editorconfig` ?